container-images/tcib/base/openstackclient/openstackclient.yaml

```diff
@@ -16,11 +16,16 @@ tcib_packages:
   - python3-heatclient
   - python3-ironicclient
   - python3-manilaclient
-  - python3-observabilityclient
   - python3-octaviaclient
   - python3-aodhclient
   - bash-completion
   - iputils
+  #rhel: We can uncomment this when we have rhel specific pkgs
+  x86_64:
+  - python3-observabilityclient
+  el8:
```

Comment:
how/where are these 'architectures' decided. el8 is operating system specific and x86_64 is cpu architecture specific. how/where is this going to be used

Reply:
So, we had an issue with python3-observabilityclient pkg which we dont need in rhel now - https://redhat-internal.slack.com/archives/C04HN4L5XHV/p1706022079814729. So we want to follow the same pattern we followed with https://opendev.org/openstack/tripleo-common/src/branch/master/container-images/tcib/base/collectd/collectd.yaml#L50 to get this package excluded in rhel.

Reply:
ok thanks for the extra info.

as we discussed in yesterday's review call, lets keep only the one we need here (e.g. "el9" or "rhel9" or something similar) and then we will also need some further work to wire it up so that we get the right packages for rhel9

Reply:
we should not have any el8 related content we will be building this for 9.4

Reply:
observabilityclient is not built on el8, only for newer releases.